pandas/_libs/groupby.pyx

```diff
@@ -1075,6 +1075,8 @@ def group_mean(
                     y = val - compensation[lab, j]
                     t = sumx[lab, j] + y
                     compensation[lab, j] = t - sumx[lab, j] - y
+                    if compensation[lab, j] != compensation[lab, j] :
```

Comment:
nit: Can you remove the space prior to the `:`

Reply:
Done !

Reply:
@rhshadrach  I have removed the space.. can you resolve the` change request ` ?